### PR TITLE
add `D.Chisquare`

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

File: tests/chainer_tests/distributions_tests/test_chisquare.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            "sample", "support", "variance"])
+
+        k = numpy.random.randint(1, 10, self.shape).astype(numpy.float32)
+        self.params = {"k": k}
+        self.scipy_params = {"df": k}
+
+        self.support = "positive"
+
+    def sample_for_test(self):
+        smp = numpy.random.gamma(
```
Comment (toslunar) [inline tests/chainer_tests/distributions_tests/test_chisquare.py]: How did you choose the distribution?

Comment (YoshikawaMasashi) [reply tests/chainer_tests/distributions_tests/test_chisquare.py]: There is no reason other than that this is a distribution whose support is [0, ∞).
I'll change this to chisquare.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**